css/styles.css

```diff
@@ -56,6 +56,144 @@ a {
   display: none;
 }
 
+.popup {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: white;
+  border: 20px solid #c1c7d0;
+}
+
+.popupdesk {
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: white;
+}
+
+.xsign {
+  position: absolute;
+  top: 0;
+  margin-left: 298px;
+  padding-top: 50px;
+  z-index: 1;
+}
+
+.xsigndesk {
+  position: absolute;
+  right: 0;
+  margin-right: 40px;
+  z-index: 10;
+  margin-top: 60px;
+}
+
+.pop-img {
+  width: 370px;
+  position: relative;
+  height: 320px;
+  padding-right: 35px;
+}
+
+.popdesk-img {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-top: 10px;
+  width: 600px;
+  height: 350px;
+  margin-left: 360px;
+}
+
+.phead {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 260px;
+  margin-left: 10px;
+}
+
+.pheaddesk {
+  display: flex;
+  flex-direction: column;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  width: 460px;
+  margin-bottom: 10px;
+  margin-left: 10px;
+  margin-right: 420px;
+}
+
+.textdesk {
+  display: flex;
+  flex-direction: row;
+}
+
+.ptext {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.plist {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+  padding-left: 0;
+}
+
+.plistdesk {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+}
+
+.panc {
+  padding: 8px 12px;
+  background: #ebf0ee;
+  color: #3a4a42;
+  text-align: center;
+  text-decoration: none;
+  border-radius: 10px;
+  margin-right: 10px;
+}
+
+.pbuttons {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  list-style: none;
+  gap: 47px;
+}
```

Comment:
- [ ] Please I noticed that the two `buttons `are hidden for the mobile version, kindly adjust this as well. 🙏